Formula/dotnet.rb

```diff
@@ -50,7 +51,7 @@ class Dotnet < Formula
 
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
+  depends_on "python@3.11" => :build
   depends_on "icu4c"
   depends_on "openssl@1.1"
```

Comment:
Was originally going to update OpenSSL, but decided to avoid mixing OpenSSL on Linux (`krb5` uses `openssl@1.1`).